views/extension/_sidebar.php

```diff
@@ -10,7 +10,7 @@
 /** @var $category string */
 /** @var $tag \app\models\ExtensionTag */
 ?>
-<?= Html::a('<span class="big">Create</span><span class="small">new extension</span>', ['create'], ['class' => 'btn btn-block btn-new-extension']) ?>
+<?= Html::a('<span class="big">Create</span><span class="small">a new extension</span>', ['create'], ['class' => 'btn btn-block btn-new-extension']) ?>
```

Comment:
That's for adding new extension so I think original text is correct.

Reply:
Both ways are correct. "Create new extension" is shorter and you wouldn't say it, but it's appropriate for a label. "Create a new extension" is more natural. Will revert it, same as with the article one below.